cmd/integration_tests_test.go

```diff
@@ -31,31 +30,6 @@ func (m *mockIntegrationTests) CreateTestData(ctx context.Context, opts integrat
 	return m.Called(ctx, opts).Error(0)
 }
 
-func Test_IntegrationTestsCommand_Command(t *testing.T) {
```

Comment:
This test is impossible to fix because `create-data` now requires an admin server running